drivers/debug/Kconfig.rtt

```diff
@@ -74,6 +74,9 @@ config SEGGER_RTT_SECTION_NONE
 config SEGGER_RTT_SECTION_DTCM
 	bool "Place RTT data in the DTCM linker section"
 
+config SEGGER_RTT_SECTION_FIXED_ADDR
+	bool "Place RTT data in specific linker section with fixed address"
```

Comment:
don't we want `default y`? If bootloader is not used then it has no difference and when bootload is used then probably most users would like to get logs from bootloader and application.

Reply:
Agree, done.

Reply:
I've added a `if CPU_CORTEX_M`, thanks.